osu.Game/OsuGameBase.cs

```diff
@@ -299,16 +299,23 @@ private void load(ReadableKeyCombinationProvider keyCombinationProvider)
 
             GlobalActionContainer globalBindings;
 
-            var mainContent = new Drawable[]
+            base.Content.Add(new SafeAreaContainer
             {
-                MenuCursorContainer = new MenuCursorContainer { RelativeSizeAxes = Axes.Both },
-                // to avoid positional input being blocked by children, ensure the GlobalActionContainer is above everything.
-                globalBindings = new GlobalActionContainer(this)
-            };
-
-            MenuCursorContainer.Child = content = new OsuTooltipContainer(MenuCursorContainer.Cursor) { RelativeSizeAxes = Axes.Both };
-
-            base.Content.Add(CreateScalingContainer().WithChildren(mainContent));
+                SafeAreaOverrideEdges = Edges.Bottom,
```

Comment:
I'd prefer if this was iOS only and/or explicitly marked why this is required.

Reply:
Adding to this, if the bottom safe area on iOS is always "wrong", why not have it always be `0` in [`IOSGameWindow`](https://github.com/ppy/osu-framework/blob/99786238a02e0d6b69da86dd52e5506ee8ec0566/osu.Framework.iOS/IOSGameWindow.cs#L64)?

Reply:
I'd rather keep things simple as long as we can. Are there any examples on android where the bottom edge is unsafe to overflow into?

Reply:
Any tablet with a notch rotated upside down (eg. [Samsung Galaxy Fold](https://user-images.githubusercontent.com/16479013/152507786-76cf4354-e510-4a48-9ccb-3b13c12db997.png)), or any phone with a notch rotated upside down (if we were to allow such orientation in the future) are the plausible examples.

And also [this](https://www.slashgear.com/sharp-aquos-r2-compact-double-notch-android-smartphone-15553838/).

Reply:
Will make it only apply to apple devices for now. Also what an ugly contraption that first example is..